browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchFragment.java

```diff
@@ -158,8 +163,19 @@ private boolean isReturnUrlSetup() {
                 .addCategory(Intent.CATEGORY_DEFAULT)
                 .addCategory(Intent.CATEGORY_BROWSABLE);
 
-        List<ResolveInfo> activities = mContext.getPackageManager()
+        List<ResolveInfo> availableActivities = availableActivities(intent);
+
+        return availableActivities != null && availableActivities.size() == 1;
```

Comment:
I'm curious on why we are checking for exactly one activity here instead of `>0`

Reply:
I think we expect there to be one and only one Activity that responds to the return url. I'm basing it off of the [error message we return](https://github.com/braintree/browser-switch-android/blob/master/browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchFragment.java#L128), though.

Reply:
Oh that makes sense 👍